apps/view/src/Nav/OpenFileDrawer.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import {Formik, Form, Field} from 'formik'
+import {Button, Drawer, Icon, Label, getButtonStyle} from '../ui'
+import {FileEvent} from '../types'
+
+const TITLE = 'new board'
+
+const UPLOAD_FROM_COMPUTER_LABEL = 'upload files from your computer'
+const ENTER_URL_LABEL = 'enter the URL of a ZIP archive'
+const ENTER_URL_PLACEHOLDER = 'https://tracespace.io/view/arduino-uno.zip'
```

Comment:
Need to replace this constant with the same logic that's used in the welcome screen URL input